src/App.js

```diff
@@ -22,11 +23,17 @@ class App extends Component {
     return (
       <Router>
         <div>
-          <Banner />
-          <I18nBanner />
+          <Route path="/" render={props => (
```

Comment:
in order to give the Header component `props.location`, I had to wrap it in a Route component. We'll see how this may need to change in an SSR world.